packages/aws-cdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Running the tests
```

Comment:
I think this should go in the main CONTRIBUTING document, so it will be visible and easy to find. You could link to it from the README file of aws-cdk

Reply:
It seems weird to me to put instructions on a particular package far away from it. But I will definitely link from the main `CONTRIBUTING` file to this.